src/Microsoft.Fx.Portability.Offline/Data.cs

```diff
@@ -63,6 +64,20 @@ public static IEnumerable<BreakingChange> LoadBreakingChanges()
             }
         }
 
+        private static IEnumerable<string> GetLocalAllowedCategories()
+        {
+            // Check to see if a file defining valid categories exists
+            var categoriesPath = Path.Combine(GetCurrentDirectoryName(), "BreakingChanges", "BreakingChangeCategories.json");
+            if (File.Exists(categoriesPath))
+            {
+                using (var categoriesFile = File.Open(categoriesPath, FileMode.Open, FileAccess.Read))
+                {
+                    return categoriesFile.Deserialize<string[]>();
+                }
+            }
+            return null;
```

Comment:
I think it's better to return null if there is no AllowedCategories file because if we pass a non-null list of allowed categories to BreakingChangeParser.FromMarkdown, then all categories used will be expected to be in that enumerable.

In other words, passing null to BreakingChangeParser.FromMarkdown will allow any categories (which I think is what we want if the file is missing) whereas passing Enumerable.Empty won't allow any categories.

I suppose we could add some logic to BreakingChangeParser.FromMarkdown to understand that an empty list really means any breaking changes are allowed, but the current design felt more natural and does allow for the (probably useless) corner case of saying 'no breaking changes should have categories'.